common/scala/src/main/scala/whisk/core/connector/Message.scala

```diff
@@ -53,7 +54,8 @@ case class ActivationMessage(
     activationId: ActivationId,
     activationNamespace: EntityPath,
     content: Option[JsObject],
-    cause: Option[ActivationId] = None)
+    cause: Option[ActivationId] = None,
+    traceMetadata: Option[SpanMetadata] = None)
```

Comment:
Are you suggesting to not include the tracing metadata in the data plane or just be mindful about the size ? 
If we don't include it, do we have a way around it without affecting latency ? If it had been an HTTP request, then the parent span ID would have been sent as a header; in this case, with Kakfa, it's sent similarly; headers are like the `metadata` IMO.